ui/src/views/infra/zone/ZoneWizardPhysicalNetworkSetupStep.vue

```diff
@@ -129,24 +130,26 @@
       :visible="showError"
       :title="`${$t('label.error')}!`"
       :maskClosable="false"
-      :okText="$t('label.ok')"
-      :cancelText="$t('label.cancel')"
-      @ok="() => { showError = false }"
+      :closable="true"
+      :footer="null"
       @cancel="() => { showError = false }"
+      v-ctrl-enter="showError = false"
       centered
     >
       <span>{{ $t('message.required.traffic.type') }}</span>
+      <div :span="24" class="action-button">
+        <a-button @click="showError = false">{{ $t('label.cancel') }}</a-button>
+        <a-button type="primary" ref="submit" @click="showError = false">{{ $t('label.ok') }}</a-button>
+      </div>
     </a-modal>
     <a-modal
       :title="$t('label.edit.traffic.type')"
       :visible="showEditTraffic"
       :closable="true"
       :maskClosable="false"
-      :okText="$t('label.ok')"
-      :cancelText="$t('label.cancel')"
-      @ok="updateTrafficLabel(trafficInEdit)"
-      @cancel="cancelEditTraffic"
       centered
+      :footer="null"
+      v-ctrl-enter="updateTrafficLabel(trafficInEdit)"
```

Comment:
`TypeError: trafficInEdit is null`. Looks like an error elsewhere in the file, can you fix it ?